<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLite3 editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>db_ic.ico</normaloff>db_ic.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>131</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Open data base</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_row_box">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Add row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_row_box">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reselect_box">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reselect table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="request">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Make request</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submit_box">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="revert_box">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>851</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>10</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>CHANGE TABLE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>10</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>FILTER PATTERN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="dialog_search">
         <property name="font">
          <font>
           <family>Mathcad UniMath</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Boutton_clear">
         <property name="font">
          <font>
           <family>Mathcad UniMath</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>851</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>12</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>BASE TABLE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Mathcad UniMath</family>
         <pointsize>12</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>SORT TABLE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionby_id">
   <property name="text">
    <string>by id</string>
   </property>
  </action>
  <action name="actionby_model">
   <property name="text">
    <string>by model</string>
   </property>
  </action>
  <action name="actionOpen_DB">
   <property name="text">
    <string>Open DB</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
